res/code_templates/relationship-right/custom/Extension/modules/__module__/Ext/Vardefs/idx_for___relationship-right__.php.twig

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * @author SugarCli
+ * SugarCli created file
+ */
+
+// Relationship foreign key to {{ moduleLeft|replace({'_': ' '}) }} index
+$dictionary["{{ moduleBean }}"]["indices"][] = array(
```

Comment:
* Should we consider ability to add length for an index as optional parameter from CLI tool or it should be a manual job by tweaking the index definition file?
* Tried to define index on multiple columns but the command is putting encoded string in the fields key of index definition.